source/Ubuntu-B.ufo/data/com.github.fonttools.ttx/T_S_I__1.ttx

```diff
@@ -194,8 +194,8 @@ IUP[Y]
 /* Composite glyph autohinted */
 
 USEMYMETRICS[]
-OFFSET[R],161,0,0
-OFFSET[R],143,485,160
+OFFSET[R], 161, 0, 0
+OFFSET[R], 143, 485, 160
```

Comment:
The `OFFSET[R]` whitespace changes are diff noise, and ideally should be filtered out from the actual commit.